src/NoteWorkspace.ts

```diff
@@ -160,7 +160,8 @@ export class NoteWorkspace {
 
   static slugifyTitle(title: string): string {
     return title
-      .replace(/\W+/gi, this.slugifyChar()) // non-words to hyphens (or underscores)
+      .normalize("NFD").replace(/[\u0300-\u036f]/g, "") // Remove accents
```

Comment:
You know, I was originally thinking we do something like this, but I am now thinking we just remove all of the special characters. So how about we just nix the `normalize` call and add one more the test to assert that:

`noteFileNameFromTitle("Šömè-Títlê") == ``"šömè-títlê"`

I think that is prolly best. Want to do that and I'll merge?

Reply:
Cool, i'll try that later today